### ISSUE TITLE
Spread operator not working.

### ISSUE THREAD

**Issue by sorahn** [maxogden/standard-format#145]
Doesn't look like the spread operator syntax is very happy right now.

```javascript
function API_URL (base, endpoint, filters = {}) {
  return base.concat(endpoint, '?', serialize({...filters, json: 1}))
}
```
results in: `SearchActions.js: Error: Line 29: Unexpected token ...`

**Comment by bcomnes** [comment]
The formatter basically doesn't have support for ES6 yet.  We hope to correct this soon.

**Comment by sorahn** [comment]
Yeah, I Think I found out after some more googling last night that this an espree issue.  Thanks for the quick reply. :)

**Comment by yoshuawuyts** [comment]
It appears the object rest spread [was added](https://github.com/millermedeiros/esformatter/commit/f28b0dad45515e9a2a9c957cd06341d9f94ad9db) for the `v0.8.0` milestone in Sept 2015, it doesn't appear to be working on more complicated uses tho (rest returns, wonky array things, etc.)

**Comment by millermedeiros** [comment]
just released `esformatter-semicolon-first@1.2.0` with the fix!

**Comment by feross** [comment]
Nice!
